.buildkite/longruns/pipeline.yml

```
@@ -39,25 +39,13 @@ steps:
       # TODO: Add all milestone long simulation runs
 
       - label: ":computer: held suarez (ρe)"
-        command: "julia --color=yes --project=examples --threads=8 examples/hybrid/driver.jl --TEST_NAME held_suarez_rhoe --forcing held_suarez --t_end 103680000 --job_id longrun_hs_rhoe --dt_save_to_sol 8640000" # 1200 days
-        artifact_paths: "longrun_hs_rhoe/*"
-        agents:
-          slurm_cpus_per_task: 8
-
-      - label: ":computer: held suarez (ρe_int)"
-        command: "julia --color=yes --project=examples --threads=8 examples/hybrid/driver.jl --TEST_NAME held_suarez_rhoe_int --forcing held_suarez --t_end 103680000 --job_id longrun_hs_rhoeint --dt_save_to_sol 8640000" # 1200 days
-        artifact_paths: "longrun_hs_rhoeint/*"
-        agents:
-          slurm_cpus_per_task: 8
-
-      - label: ":computer: held suarez (ρe)"
-        command: "julia --color=yes --project=examples --threads=8 examples/hybrid/driver.jl --TEST_NAME held_suarez_rhoe --forcing held_suarez --t_end 103680000 --job_id longrun_hs_rhoe_largerdt --dt_save_to_sol 21600" # 1200 days
+        command: "julia --color=yes --project=examples --threads=8 examples/hybrid/driver.jl --TEST_NAME held_suarez_rhoe --forcing held_suarez --t_end 103680000 --upwinding none --fps 30 --job_id longrun_hs_rhoe_largerdt --dt_save_to_sol 21600" # 1200 days
```

```suggestion
        command: "julia --color=yes --project=examples --threads=8 examples/hybrid/driver.jl --TEST_NAME held_suarez_rhoe --forcing held_suarez --t_end 103680000 --upwinding none --fps 30 --job_id longrun_hs_rhoe --dt_save_to_sol 21600" # 1200 days
```
I think it's just standard dt (400s)?

---

Huh, yeah, I think that was when I was testing the larger dt, I forgot to switch it back, thanks!